python-toolbox/marvin_python_toolbox/management/engine.py

```diff
@@ -474,6 +474,56 @@ def generate(name, description, mantainer, email, package, dest, no_env, no_git,
             shutil.rmtree(dest)
 
 
+
+@cli.command('engine-delete', help='Delete an existing marvin engine project.')
+@click.option('--name', '-n', prompt='Project name', help='Project name')
+@click.option('--dest', '-d', envvar='MARVIN_HOME', type=click.Path(exists=True), help='Root folder path for the creation')
+def delete_engine(name,dest):
+    type_ = 'python-engine'
+    # Process package name
+    package = _slugify(name)
+
+    # Make sure package name starts with "marvin"
+    if not package.startswith('marvin'):
+        package = 'marvin_{}'.format(package)
+
+    # Remove "lib" prefix from package name
+    if type_ == 'lib' and package.endswith('lib'):
+        package = package[:-3]
+    # Custom strip to remove underscores
+    package = package.strip('_')
+
+    # Append project type to services
+
+    if type_ in TEMPLATE_BASES and not package.endswith('engine'):
+        package = '{}_engine'.format(package)
+
+    # Process directory/virtualenv name
+
+    # Directory name should use '-' instead of '_'
+    dir_ = package.replace('_', '-')
```

Comment:
Since we are applying the same naming process as line 410, should we make it a function so that generate and delete will always point to the same environment?

Reply:
Is truth. I will write this function.